internal/hcl/parser.go

```diff
@@ -189,8 +215,21 @@ func (p *Parser) ParseDirectory() (*Module, error) {
 		nil,
 		p.workspaceName,
 		p.blockBuilder,
+		p.newSpinner,
 	)
 
+	if v := evaluator.MissingVars(); len(v) > 0 {
+		if p.writeWarning != nil {
+			p.writeWarning(
+				fmt.Sprintf(
+					"Input values were not provided for the following Terraform variables: %s. %s",
+					strings.TrimRight(strings.Join(v, ", "), ", "),
+					"Use --terraform-var-file or --terraform-var to specify input vars.",
```

Comment:
```suggestion
					"Input values were not provided for following Terraform variables: %s. %s",
					strings.TrimRight(strings.Join(v, ", "), ", "),
					"Use --terraform-var-file or --terraform-var to specify them.",
```

(just cutting a few words to make the sentence shorter, where possible we omit "the" in our CLI help text outputs but maybe I'm going too far)